pkgs/top-level/python-packages.nix

```
@@ -13510,6 +13510,10 @@ self: super: with self; {
 
   quaternion = callPackage ../development/python-modules/quaternion { };
 
+  qubes-vmm-pyxen = toPythonModule (pkgs.qubes-vmm-xen.override {
```

```suggestion
  qubes-vmm-xen = toPythonModule (pkgs.qubes-vmm-xen.override {
```

py prefix was there because I wasn't sure how to access non-overidden packages from python module.

I don't like name collisions here, but it seems to be norm (e.g see hwdata/python3Packages.hwdata)